frontend/src/app/core/state/days/day.service.ts

```diff
@@ -39,17 +39,22 @@ export class DayResourceService extends ResourceCollectionService<IDay> {
     return this
       .requireNonWorkingYear$(input)
       .pipe(
-        map((days) => days.findIndex((day:IDay) => !day.working && day.date === date) !== -1),
+        map((days) => days.findIndex((day:IDay) => day.date === date) !== -1),
       );
   }
 
+  public isNonWorkingDay(date:Date):boolean {
+    let isNonWorkingDay = false;
+    this.isNonWorkingDay$(date).subscribe((isNonWorking) => { isNonWorkingDay = isNonWorking; });
```

Comment:
This will not work as such, if you have to do a request for `isNonWorkingDay$`. We'll need to alter the calling side to consume an observable, or a promise